### ISSUE TITLE
вопрос

### ISSUE THREAD

**Issue by nikimagic** [DJAlexJ/RL-policygradient#4]
Я правильно понимаю,  что мы собираем несколько траекторий, потом немного с ними  колдуем и получаем веса  для сети ?

**Comment by XuMuK1** [comment]
> Я правильно понимаю,  что мы собираем несколько траекторий, потом немного с ними  колдуем и получаем веса  для сети ?

Да, именно так, только градиенты, а не веса. Вообще в теории эту идею можно легко перенести и на меньшую длину траектории или вовсе на траекторию длиной в один шаг.
